synapse/lib/stormlib/gen.py

```diff
@@ -42,6 +42,37 @@ class LibGen(s_stormtypes.Lib):
                       {'name': 'cve', 'type': 'str', 'desc': 'The CVE id.'},
                   ),
                   'returns': {'type': 'storm:node', 'desc': 'A risk:vuln node with the given CVE.'}}},
+
+        {'name': 'riskThreat', 'desc': 'Returns risk:vuln node by CVE, adding the node if it does not exist.',
```

Comment:
Copypasta. Suggest "Returns risk:threat node based on the threat and reporter names, adding the node if it does not exist."